assembl/static2/tests/unit/reducers/postsReducer.spec.js

```diff
@@ -0,0 +1,128 @@
+import * as reducers from '../../../js/app/reducers/postsReducer';
+
+describe('Posts reducers', () => {
+  describe('topPostSubject reducer', () => {
+    const { topPostSubject } = reducers;
+    it('should return the initial state', () => {
+      expect(topPostSubject(undefined, {})).toEqual('');
+    });
+
+    it('should return state by default', () => {
+      const state = 'New subject';
+      const expected = 'New subject';
+      const actual = topPostSubject(state, {});
+      expect(actual).toEqual(expected);
+    });
+
+    it('should handle UPDATE_TOP_POST_SUBJECT action type', () => {
+      const state = 'New subject';
+      const action = {
+        type: 'UPDATE_TOP_POST_SUBJECT',
+        topPostSubject: 'New subject'
+      };
+      const actual = topPostSubject(state, action);
+      const expected = 'New subject';
+      expect(actual).toEqual(expected);
+    });
+  });
+
+  describe('topPostBody reducer', () => {
+    const { topPostBody } = reducers;
+    it('should return the initial state', () => {
+      expect(topPostBody(undefined, {})).toEqual('');
+    });
+
+    it('should return state by default', () => {
+      const state = 'New body';
+      const expected = 'New body';
+      const actual = topPostBody(state, {});
+      expect(actual).toEqual(expected);
+    });
+
+    it('should handle UPDATE_TOP_POST_BODY action type', () => {
+      const state = 'New body';
+      const action = {
+        type: 'UPDATE_TOP_POST_BODY',
+        topPostBody: 'New body'
+      };
+      const actual = topPostBody(state, action);
+      const expected = 'New body';
+      expect(actual).toEqual(expected);
+    });
+  });
+
+  describe('topPostFormStatus reducer', () => {
```

Comment:
Same here, I find these tests not really useful, they could be autogenerated or replaced by better ones.